main.tex

```diff
@@ -381,9 +383,8 @@ \section{Reflection about Reflection: an Impossibility Result}
 
 \subsection{The Inner Reflection Principle}
 The reflection principle is stated in the metalanguage, and is a 
-\emph{statement which is true about $\PP$},
-not an axiom in $T$ to which $\PP$ assigns probability 1.
-In contrast to this principle, which we can think of as an \emph{outer reflection principle}, we might also want $\PP$ to assign probability 1 to $P$'s reflective consistency.
+\emph{statement which is true about $\PP$}, not a sentence in the language.
+In contrast to this metalanguage reflection principle, which we can think of as an \emph{outer reflection principle}, we might also want $\PP$ to assign probability 1 to $P$'s reflective consistency.
```

Comment:
take out "metalanguage" in "metalanguage reflection principle".
also maybe less \emph